compiler/include/flags_list.h

```diff
@@ -96,6 +96,7 @@ symbolFlag( FLAG_EXPORT_INIT, ypr, "export init", "indicate that the module's in
 symbolFlag( FLAG_EXPR_TEMP , npr, "expr temp" , "temporary that stores the result of an expression" )
 symbolFlag( FLAG_EXTERN , npr, "extern" , "extern variables, types, and functions" )
 symbolFlag( FLAG_FAST_ON , npr, "fast on" , "with FLAG_ON/FLAG_ON_BLOCK, \"on block\" , use fast spawning option (if available)" )
+symbolFlag( FLAG_FAST_ON_SAFE_EXTERN, ypr, "fast-on safe extern function", "extern function is safe for fast-on optimization")
```

Comment:
Since I don't want this to apply to regular Chapel procedures, I wanted to make it clear that this is only for extern functions.  I'm open to changing the name, but would like to keep that property.